Stratis.Bitcoin.Features.BlockStore/LoopSteps/ProcessPendingStorageStep.cs

```diff
@@ -189,12 +205,27 @@ internal ProcessPendingStorageContext(BlockStoreLoop blockStoreLoop, ChainedBloc
         internal ConcurrentStack<BlockPair> PendingBlockPairsToStore = new ConcurrentStack<BlockPair>();
 
         /// <summary>
-        /// Gets the next block to process.
+        /// Break execution if:
+        /// <list>
+        ///     <item>1: At the tip</item>
+        ///     <item>2: Block is already in store or pending insertion</item>
+        /// </list>
         /// </summary>
-        internal void GetNextBlock()
+        internal bool CanProcessNextBlock()
```

Comment:
return value comment missing and SENTENCES

Reply:
Will fix